webpack.config.js

```diff
@@ -8,15 +8,15 @@ module.exports = {
     path: path.join(__dirname, '/dist'),
     filename: 'bundle.js',
   },
+  resolve: {
+    extensions: ['', '.jsx', '.js', '.json']
+  },
   module: {
     loaders: [
       {
-        test: /\.jsx?$/,
+        test: /(\.js|\.jsx)$/,
```

Comment:
so it checks out .js and .jsx files. this might have been the import issue you had earlier.